joerd/output/skadi.py

```diff
@@ -108,13 +109,18 @@ def render(self, tmp_dir):
 
         composite.compose(self, dst_ds, logger, min(dst_x_res, dst_y_res))
 
-        logger.info("Writing SRTMHGT: %r" % outfile)
+        logger.debug("Writing SRTMHGT: %r" % hgt_file)
```

Comment:
The change from `info` to `debug` here was just for consistency. There's still an `info`-level message below once the tile has been successfully generated.